server/svix-server/src/v1/endpoints/endpoint/crud.rs

```diff
@@ -87,7 +87,7 @@ async fn create_endp_from_data(
         let txn = ctx!(db.begin().await)?;
```

Comment:
Why does app (above) uses the `transaction!` macro and this one doesn't?

Also, does it happen at the transaction commit, or just when you insert inside of the transaction?

Also also, if it's at the end of the transaction then I think the above (app) is still broken? If it's not, then this change below is superfluous?

Reply:
I'll check whether the error happens on insert or on committing the transaction

Reply:
It happens inside the transaction, not on commit. I removed what was unnecessary.